test/utils-tests.jl

```diff
@@ -42,6 +42,17 @@ end
          [44.38225395397647, 48.542947077386664, 67.33061196497327])
 end
 
+# Test baryvel
+@testset "baryvel" begin
+    dvelh_o, dvelb_o = @inferred(baryvel(AstroLib.J2000, 1950))
+    @test dvelh_o ≈ [-29.858889109661163, -4.685067254882613, -2.0307823051175067]
```

Comment:
For example, I quickly tried to debug the function, `pertl` value looks to be wrong.